text/0000-fabric-private-chaincode-1.0.md

```diff
@@ -329,14 +329,14 @@ This prevents a malicious peer from responding with a wrong transaction encrypti
 
 
 ### Enclave Endorsement Validation
-The Enclave Endorsement Validation component verifies the correctness of a result of an FPC Chaincode execution and persists state updates to the ledger.
```

Comment:
I guess this section should start like
"This component implements the second step of an FPC chaincode invocation."